Source/Utils/ParticleUtils.H

```diff
@@ -31,6 +32,96 @@ namespace ParticleUtils {
     findParticlesInEachCell( int const lev, amrex::MFIter const& mfi,
                              WarpXParticleContainer::ParticleTileType const& ptile);
 
+    /**
+     * \brief Return (relativistic) particle energy given velocity and mass.
+     * Note the use of `double` since this calculation is prone to error with
+     * single precision.
+     *
+     * @param[in] v2 square of particle speed ( v dot v)
+     * @param[in] mass particle mass
+     * @param[out] energy particle energy in eV
+     */
+    AMREX_GPU_HOST_DEVICE AMREX_INLINE
+    void getEnergy ( amrex::ParticleReal const v2, double const mass,
```

Comment:
I have used the notation that `u = gamma*v`, so the argument should be `u2`. Also, the comment should make it clear that this is `gamma*v`.